### DISCUSSION TITLE
use custom PKGBUILD with paru

### DISCUSSION THREAD

**Discussion by amtoine** [Morganamilo/paru#869] [Q&A]
hello there :wave: :yum: 

i have a bunh of `PKGBUILD`s for my ArchLinux config [here](https://github.com/goatfiles/pkgbuilds/tree/main/x86_64)
currently, i have a script called `install.sh` and i need to manually install the packages with `./install.sh` from within the root of the repo...
this works fine but the packages are not tracked by `pacman` and it's quite tideous to do each time one of my own packages needs an update :thinking: 

i could use the AUR and push all these `PKGBUILD`s to the repository, but i'm wondering if there is an easier solution which could directly use my `PKGBUILD`s :yum: 

**how can i achieve to tell `paru` to use https://github.com/goatfiles/pkgbuilds/tree/main/x86_64 as well as https://aur.archlinux.org/?**
maybe i **missed some piece of documentation?** :open_mouth: 

cheers :tada: :wink:

**Comment by Morganamilo** [comment]
See https://github.com/Morganamilo/paru/issues/585#issuecomment-1236191380

I'm hoping to get v2 out before the end of the year. Though I'm moving right now and don't have a PC set up to work on.

**Reply by amtoine** [reply]
ooooh so that's something for the future `v2` version of `paru` :open_mouth: 

that would be really cool :relieved: :star_struck:
